### PR TITLE
chore: Run CI on default runner

### PR DESCRIPTION
**Pull request by mediuminvader**

### CODE REVIEW

File: .github/workflows/coverage.yaml
Comment (chubei) [inline .github/workflows/coverage.yaml:1]: Runner is not changed?

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 name: Dozer E2E Test
 
 on:
-  pull_request_target:
-    branches: [main]
-  workflow_dispatch:
+  schedule:
```
Comment (chubei) [inline .github/workflows/e2e.yaml:4]: Don't need to run on `merge_group` if we're running nightly? And `workflow_dispatch` can be kept so it's easier to test

**Comment by chubei** [comment]
`ci.yaml` runner is not changed?

File: .github/workflows/coverage.yaml
Comment (chubei) [inline .github/workflows/coverage.yaml:1]: Does `group: Org Larger Runners` work?

File: .github/workflows/dozer.yaml
Comment (chubei) [inline .github/workflows/dozer.yaml:1]: Does `group: Org Larger Runners` work?

**Review by chubei — APPROVED** [review]
I don't know if group affects anything. How did you test?

File: .github/workflows/dozer.yaml
Comment (mediuminvader) [reply .github/workflows/dozer.yaml:1]: should just remove, right?

Comment (chubei) [reply .github/workflows/dozer.yaml:1]: Probably. Have to test

**Comment by chubei** [comment]
@mediuminvader are you still working on it? `coverage.yaml` and the workflows in `dozer-cloud` are still using larger runners.

**Comment by chubei** [comment]
Replaced by #1910

**Pull request closed without merging**